presto-main/src/main/java/io/prestosql/execution/QueryManagerConfig.java

```diff
@@ -60,6 +61,7 @@
     private Duration queryMaxRunTime = new Duration(100, TimeUnit.DAYS);
     private Duration queryMaxExecutionTime = new Duration(100, TimeUnit.DAYS);
     private Duration queryMaxCpuTime = new Duration(1_000_000_000, TimeUnit.DAYS);
+    private DataSize queryMaxScanPhysicalBytes;
```

Comment:
Change to `Optional<DataSize>` and make the default `Optional.empty()`

Reply:
Done